test/jest/Pool.spec.ts

```diff
@@ -157,7 +157,7 @@ describe('P2P Pool', () => {
     expect(sessionInitPacket).toBeTruthy();
   });
 
-  test('it rejects when already connected', async () => {
+  test.skip('it rejects when already connected', async () => {
```

Comment:
This will conflict with #1318, but I understand the need for it to cut down on the number of retries needed in Travis. I'm going to try again now to get to the bottom of this test case so hopefully we can merge the fix rather than disable the test, but I also don't want that to hold up this PR if I can't fix it very soon.